android/source/VideoLocker/src/org/edx/mobile/player/PlayerFragment.java

```diff
@@ -1112,30 +1113,32 @@ private void setClosedCaptionData(Caption text){
             TextView subTitlesTv = (TextView) getActivity().findViewById(R.id.txtSubtitles_tv);
             if(subTitlesTv!=null ){
                 if(text!=null){
+                    int twenty_dp = (int) UiUtil.getParamsInDP(getResources(),20);
+                    int ten_dp = (int) UiUtil.getParamsInDP(getResources(),20);
```

Comment:
This says ``20``, but the variable is named ``ten``.

Also, maybe we should give the variables semantic names, like ``full_margin`` and ``small_padding`` or something like that.